src/vivarium_csu_ltbi/components/treatment.py

```diff
@@ -0,0 +1,165 @@
+import pandas as pd
+
+from vivarium_csu_ltbi.components.names import (ACTIVETB_SUSCEPTIBLE_HIV,
+                                                ACTIVETB_POSITIVE_HIV,
+                                                LTBI_POSITIVE_HIV,
+                                                PROTECTED_TB_POSITIVE_HIV,
+                                                SUSCEPTIBLE_TB_POSITIVE_HIV)
+
+
+# noinspection PyAttributeOutsideInit
+class LTBITreatmentCoverage:
+
+    @property
+    def name(self):
+        return 'ltbi_treatment_coverage'
+
+    def setup(self, builder):
+        self.time = builder.time.clock()
+
+        self.treatment_stream = builder.randomness.get_stream(f'{self.name}.treatment_selection')
+        self.adherence_stream = builder.randomness.get_stream(f'{self.name}.adherence_propensity')
+
+        self.household_tb_exposure = builder.value.get_value('household_tuberculosis.exposure')
+
+        # NOTE: Just write these into the same key, later
+        three_hp_adherence_data = builder.data.load("three_hp.adherence")
+        six_h_adherence_data = builder.data.load("six_h.adherence")
+        self.adherence = builder.lookup.build_table(pd.concat([three_hp_adherence_data, six_h_adherence_data], axis=0),
+                                                    parameter_columns=['age', 'year'],
+                                                    key_columns=['sex', 'treatment_type'],
+                                                    value_columns=['value'])
+
+        six_h_coverage_data = builder.data.load("six_h.coverage.proportion")
+        self.six_h_with_hiv, self.six_h_under_five_hhtb = self.setup_coverage_tables(builder, six_h_coverage_data)
+        three_hp_coverage_data = builder.data.load("three_hp.coverage.proportion")
+        self.three_hp_with_hiv, self.three_hp_under_five_hhtb = self.setup_coverage_tables(builder,
+                                                                                           three_hp_coverage_data)
+
+        self.coverage = builder.value.register_value_producer('ltbi_treatment.coverage', source=self.get_coverage,
```

Comment:
You need to specify pipeline and column dependencies here (and randomness streams if you use them).

Reply:
Same with the treatment status producer

Reply:
Let me know if it's unclear what to do and we'll talk it through.